packages/api/src/routes-v1/metrics.js

```diff
@@ -0,0 +1,100 @@
+import { secrets, database } from '../constants.js'
+import { DBClient } from '../utils/db-client.js'
+
+const db = new DBClient(database.url, secrets.database)
+
+export async function getUserMetrics() {
+  const query = db.client.from('account')
+  const res = await query.select('*', { head: true, count: 'exact' })
```

Comment:
do we need the count to be exact ? maybe we should measure how long does the other modes take with a full db

Reply:
I think for users it is fine for now but I was thinking the same thing for uploads. Lets see how long it takes and we can switch to fuzzy count if problematic.